pkg/container/types/types.go

```diff
@@ -566,6 +568,9 @@ func (t T) ToType() Type {
 	case T_varchar:
 		typ.Size = VarlenaSize
 		typ.Width = MaxVarcharLen
+	case T_embedding:
+		typ.Size = VarlenaSize
+		typ.Width = MaxEmbeddingDimension
```

Comment:
This should be a dynamic value

Reply:
but for varchar, we have MaxVarcharLen. I think `typ.Width` value will be overwritten by the user input, when he specifies it.